src/Microsoft.Health.Fhir.R4.Web/Microsoft.Health.Fhir.R4.Web.csproj

```diff
@@ -5,6 +5,24 @@
     <RootNamespace>Microsoft.Health.Fhir.Web</RootNamespace>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="hl7.fhir.r4.core-4.0.1.tgz" />
+    <None Remove="hl7.fhir.r4.elements-4.0.1.tgz" />
+    <None Remove="hl7.fhir.r4.expansions-4.0.1.tgz" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="hl7.fhir.r4.core-4.0.1.tgz">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="hl7.fhir.r4.elements-4.0.1.tgz">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="hl7.fhir.r4.expansions-4.0.1.tgz">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
```

Comment:
Should we be checking these into OSS?

Reply:
This is a good question.  Previously we did reference the specification.zip file, which was part of the nuget that we were using.  This is the same content, but in this case we are including the package in our repo.